src/Control/Email/Email.php

```diff
@@ -706,30 +719,39 @@ public function setBody($body)
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::html()
+     *
      * @return $this
      */
     public function invalidateBody()
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::html()');
         $this->setBody(null);
 
         return $this;
     }
 
     /**
+     * @deprecated 4.12.0 Will be added to return value of getData()
```

Comment:
```suggestion
     * @deprecated 4.12.0 Use getData() instead
```

Reply:
For anyone seeing this in the future: We discussed offline and decided deprecation notices shouldn't say _when_ things will be removed, in case we don't end up removing them in that release for whatever reason.